contracts/cosmwasm-vm/cw-xcall/src/send_call_message.rs

```diff
@@ -76,9 +76,10 @@ impl<'a> CwCallService<'a> {
                         } else {
                             vec![]
                         };
+                        let address = deps.api.addr_validate(r)?;
 
                         self.call_connection_send_message(
-                            &r.to_string(),
+                            &address,
```

Comment:
redundant check

Reply:
This is separate for default connection check and different protocols check, so both are needed, it will be redundant for only default connection one